wireapp/wire-ios
# chore: remove phone change flow SQCORE-1295

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQCORE-1295" title="SQCORE-1295" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />SQCORE-1295</a>  [iOS] Remove phone registration flow
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

In https://github.com/wireapp/wire-ios/pull/5665 we removed the phone registration flow, since backend no longer supports new phone registrations. But I forgot to remove the flow to change or add a phone number for an existing account, which is also blocked.

### Solutions

Make the  _Settings > Account > Phone_ setting a get only property. If the self user does not have a phone number, then the will not see this property at all. If they do, then they will see it, but can no longer change it.

### Testing

#### How to Test

- Log in as a personal user with no phone number, verify that the phone setting is not visible
- Log in as a personal user with with an existing phone number, verify that the phone setting is visible, but not mutable

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
